Makefile

```diff
@@ -5,13 +5,24 @@ export GIT_DIFF ?= off
 .PHONY: help lint lint-fix build test-unit test-integration test-integration-full deploy clean clean-deploy
 
 help:
-	echo "Please choose one of the following targets: lint, lint-fix, build, test-unit, test-integration, test-integration-full, deploy, clean, clean-deploy"
+	echo "Please choose one of the following targets: lint, lint-fix, build, test-unit, test-integration, test-integration-dry, deploy, clean, clean-deploy, serialize_package"
+
+lint lint-fix build test-unit test-integration-dry deploy clean clean-deploy serialize_package:
 
-lint lint-fix build test-unit test-integration deploy clean clean-deploy:
 	if [ "$(CLOUD)" = "bigquery" ] || [ "$(CLOUD)" = "snowflake" ]; then \
+		if [ "$@" = "serialize_package" ]; then \
+			rm -rf dist/; \
+			mkdir dist/; \
+		fi; \
 		for module in `node scripts/modulesort.js`; do \
 			echo "> Module $${module}/$(CLOUD)"; \
 			$(MAKE) -C modules/$${module}/$(CLOUD) $@ || exit 1; \
+			if [ "$@" = "serialize_package" ]; then \
```

Comment:
Why are we doing all this in the root Makefile? Couldn't we handle it in the serialize_package task in common/bigquery?

If there's a good reason to put it here I would consider having a separate target for serialize_package here even if it duplicates the `for module...` to keep the other targets simple.

Reply:
I put this target in the root Makefile in order to avoid code duplication about the module iteration. In this way, the schema of this target is in line with the already introduced (the target in common/bigquery manages just one module). However, as you said maybe an intermediate target could be sensible. Working on it.